person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
[OPTIONAL]
- [ ] As suggested by the previous reviewer it would be best you auto generate your id with the below code snippet, rather than passing it as a parameter in the initialize constructor, this would ensure that users don't get to choose their ID making it impossible for two users to have the same ID
- cose snipppet
```ruby
 @id = Random.rand(1..1000)
```
- [ ] Please kindly follow the previous reviewers suggestion on the id and implement his code snippets into your codebase :+1: